modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -440,6 +468,42 @@ Deletes a permission policy added to the specified entity.
 ----
 --
 
+[DELETE] /api/permission/policies/{kind}/{namespace}/{name}::
++
+--
+Deletes all permission policies added to the specified entity.
+
+.Request parameters
+[cols="15%,45%,15%,25%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
```

Comment:
Sugesstion (applicable gloablly):
Maybe we can change the column name from Requirement to Presence as the intent is to highlight whether this parameter must be present in the request or not. Presence can have two values -- required or optional. Requirement does not depict the correct sense here.